nuxeo-core-binarymanager-s3/src/test/resources/OSGI-INF/test-s3directupload-contrib.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+
+<component name="s3DirectUpload" version="1.0.0">
+
+  <require>org.nuxeo.ecm.core.storage.cloud.managment.contrib</require>
```

Comment:
You don't need a `<require>` in tests, as you determine the load order explicitly.